libs/react/src/lib/form/textarea/textarea.tsx

```diff
@@ -0,0 +1,80 @@
+import { IExpandableInformation, ILabelAndLabelInformation, IValidator, randomId, validateClassName } from '@sebgroup/extract';
+import { ChangeEvent, memo, useCallback, useEffect, useState } from 'react';
+import { FormItem } from '../../formItem';
+
+export interface ITextAreaProps
+  extends IExpandableInformation,
+    ILabelAndLabelInformation,
+    React.DetailedHTMLProps<React.TextareaHTMLAttributes<HTMLTextAreaElement>, HTMLTextAreaElement> {
+  label: string;
+  labelInformation?: string;
+  validator?: IValidator | undefined;
+  value?: string;
+  'data-testid'?: string;
+}
+
+export const TextArea = memo(
+  ({
+    autoComplete = 'off',
+    expandableInfo,
+    expandableInfoButtonLabel,
+    id = randomId(),
+    label,
+    labelInformation,
+    onChange,
+    role,
+    rows = 4,
+    style = {},
+    validator,
+    value,
+    "data-testid": dataTestId,
+    ...props
+  }: ITextAreaProps) => {
+    const [uuid] = useState(id);
+    const [localValue, setLocalValue] = useState(value);
+
+    useEffect(() => {
+      setLocalValue(value);
+    }, [value]);
+
+    const localOnChange = useCallback(
+      (event: ChangeEvent<HTMLTextAreaElement>) => {
+        setLocalValue(event.target.value);
+        if (onChange) onChange(event);
+      },
+      [setLocalValue, onChange]
+    );
+
+    const ariaDetails: string[] = [];
```

Comment:
added some accessibility props, since both label info and expandable info can be react elements I read that it is better to have it as aria-details instead of aria-describedBy